src/main/java/com/epam/trn/hw6/pages/MetalsAndColorsPage.java

```diff
@@ -0,0 +1,85 @@
+package com.epam.trn.hw6.pages;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Checkbox;
+import com.epam.trn.hw6.sections.MetalsAndColorsForm;
+import com.epam.trn.hw6.entities.MetalsAndColorsFormData;
+import com.epam.trn.hw6.utils.JsonDataReader;
+import org.openqa.selenium.WebElement;
+
+import static com.epam.trn.hw6.sections.MetalsAndColorsForm.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Title("Metal and Colors")
+public class MetalsAndColorsPage extends WebPage {
+
+    protected JsonDataReader jreader;
+    {
+        try {
+            jreader = new JsonDataReader();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @FindBy(css = ".form")
+    public static MetalsAndColorsForm metalsAndColorsForm;
+
+    public static void selectSummary(ArrayList<String> summary) {
```

Comment:
I don't really understand the reason of this method here. 
You have created the MetalsAndColorsForm, so everything related to this form should be performed inside of MetalsAndColorsForm class.